cafebabel/templates/base.html

```diff
@@ -55,7 +55,7 @@
 <nav>
   <ul>
     {% for lang_code, lang_label in get_languages() %}
-      <li><a href={{ url_for('cores.home_lang', lang=lang_code) }}>{{ lang_code }}</a></li>
+      <li><a href={{ url_for('cores.home_lang', lang=lang_code) }}>{{ lang_code|upper }}</a></li>
```

Comment:
No, please remove it's done on the CSS side (or remove associated CSS rules)

Reply:
i removed it on the css side dont worry, but we need to have
```
button.language {
  left: 8.5rem;
  color: #00283c;
  text-transform: uppercase;
}
```
otherwise, the button (not the menu) stays in lowercase